xbmc/video/Teletext.cpp

```diff
@@ -2225,7 +2230,7 @@ void CTeletextDecoder::RenderCharIntern(TextRenderInfo_t* RenderInfo, int Char,
 
   /* render char */
   sbitbuffer = m_sBit->buffer;
-  unsigned char localbuffer[1000]; // should be enough to store one character-bitmap...
+  unsigned char localbuffer[(m_sBit->pitch + 1) * m_sBit->height];
```

Comment:
With them makes sense, only bad about Windows, where not work with variable amount (see the Jenkins errors).
Thats was also the reason why added with 1000, instead of right.

As another solution can be a new/delete used, but then the performance must be checked.